fusioncharts/fusioncharts.d.ts

```diff
@@ -0,0 +1,302 @@
+// Type definitions for FusionCharts 3.11.2
+// Project: http://www.fusioncharts.com
+// Definitions by: Shivaraj KV <https://github.com/shivarajkv>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+
+declare namespace FusionCharts {
+
+    type ChartDataFormats = 'json' | 'jsonurl' | 'csv' | 'xml' | 'xmlurl';
+
+    type ImageHAlign = 'left' | 'right' | 'middle';
+
+    type ImageVAlign = 'top' | 'bottom' | 'middle';
+
+    interface EventObject {
+        type: string;
+
+        eventId: number;
+
+        sender: FusionCharts;
+
+        cancelled: boolean;
+
+        stopPropagation: () => void;
+
+        prevented: boolean;
+
+        preventDefault: () => void;
+
+        detached: boolean;
+
+        detachHandler: () => void;
+    }
+
+    interface ChartObject {
+
+        type?: string;
+
+        id?: string;
+
+        width?: number | string;
+
+        height?: number | string;
+
+        renderAt?: string;
+
+        dataFormat?: ChartDataFormats;
+
+        dataSource?: string | Object;
```

Comment:
Because with `{}` you won't inherit the properties from global `Object` type that you don't want and may have side effects.